python/tskit/trees.py

```diff
@@ -2917,6 +2918,68 @@ def samples(self, population=None, population_id=None):
             samples = samples[sample_population == population]
         return samples
 
+    def write_fasta(self, output, sequence_ids=None, wrap_width=60):
+        """
+        Writes a fasta formatted file from tree sequence data to the
```

Comment:
This assumes that the user knows what a fasta is. How about something like "Writes haplotype data for samples in FASTA format to the specified file-like object".